agent/grpc-external/services/peerstream/stream_test.go

```diff
@@ -423,7 +423,7 @@ func TestStreamResources_Server_StreamTracker(t *testing.T) {
 		})
```

Comment:
Changes to these tests:
1. rename some `lastRecvSuccess` to `lastRecvResourceSuccess` to match new method name
2. also test the heartbeat receive tracking